core/src/main/java/org/sonarsource/sonarlint/core/container/analysis/filesystem/InputPathCache.java

```diff
@@ -78,10 +77,6 @@ public InputDir inputDir(String relativePath) {
     return null;
   }
 
-  public InputFile inputFile(Path path) {
-    return inputFileCache.get(path);
-  }
-
   public InputDir inputDir(Path path) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "InputDir"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)